index.html

```diff
@@ -166,6 +166,15 @@ <h4 class="label">Settings</h4>
                 <button @click="reScan" class="mj-rescan">
                   Rescan
                 </button>
+              
+              <br>
```

Comment:
Using `<br>` for spacing is a bad practice. You can wrap both the field section in divs with the bottom margin as per the need.

Reply:
Should it be responsive or do I hard code a value?

Reply:
You can hard-code it.

Reply:
Please have a look @sanket143